index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tasnimul Hasan Portfolio</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+    <style>
+        body{background: url("Image/backimg.png");
+        background-repeat: no-repeat;
+        background-attachment: fixed;
+        background-size: cover;
+    }
+    </style>
```

Comment:
- [ ] Kindly move style to style file to make your code cleaner. ✌️ 
- [ ] Please after removing the code add the background to the first section because it is not the background of all body and fix it because it is not like in the  [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). You can see too that other sections have different backgrounds. 
Here you have a screenshot of how I am seeing it:

![image](https://user-images.githubusercontent.com/74506933/153657388-38312098-b66a-4513-afe8-15eae36774f6.png)